apps/cli/tf/bucket.tf

```diff
@@ -0,0 +1,73 @@
+module "acm_request_certificate" {
+  source = "cloudposse/acm-request-certificate/aws"
+
+  # https://github.com/cloudposse/terraform-aws-acm-request-certificate
+  version                           = "v0.17.0"
+  domain_name                       = "cli.sunodo.io"
+  process_domain_validation_options = true
+  ttl                               = "300"
+  zone_id                           = "Z09327832YAJGJ5APMP08"
```

Comment:
Is the `zone_id` sensitive information to be here?

Reply:
I think yes, 

It is generally recommended to treat the zone_id as sensitive information and avoid sharing it in public forums, code repositories, or any other insecure channels.

I'll fix it ...